tests/e2e/testfiles/connectivity-values.yaml

```diff
@@ -48,5 +48,6 @@
     - "Validate connectivity client from CR [1] is not installed"
     - "Enable forceRemoveDriver on CR [2]"
     - "Delete custom resource [2]"
+    - "Validate rbac deleted from namespace [test-vxflexos]"
```

Comment:
Can we check all namespaces (not just `test-vxflexos`) for the presence of the connectivity client role/bindings?

Reply:
Addressed here: https://github.com/dell/csm-operator/pull/632